doc/docs/using_dtm.md

```diff
@@ -0,0 +1,49 @@
+# Configuring and using the application
+
+In nRF Connect Direct Test Mode, you can control the reception and transmission characteristics of the Bluetooth connection through 2-wire UART, in accordance with the [Bluetooth Core specification](https://www.bluetooth.com/specifications/specs/core-specification-5-3/) (volume 6, part F).
+
+!!! info "Tip"
+     When started, nRF Connect Direct Test Mode will automatically detect if the connected device has the right firmware programmed.
+
+It is recommended to use two devices for both scenarios:
+
+* **Device 1**: One [supported device](./index.md#supported-devices) programmed with firmware compatible with Direct Test Mode.</br>
+  For example, you can use the [Direct Test Mode Bluetooth sample](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/direct_test_mode/README.html) from the nRF Connect SDK.
+
+    !!! note "Note"
+          The [Direct Test Mode Bluetooth sample](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/bluetooth/direct_test_mode/README.html) supports two Device Under Test communication protocols: 2-wire UART and experimental HCI UART interface. nRF Connect Direct Test Mode only supports the 2-wire UART.
+
+* **Device 2**: One device programmed with the firmware conformant to the Bluetooth specification.</br>
+  This can also be an off-the-shelf product certified for Bluetooth 5.3.
+
+## Transmitting
+
+To execute transmitting, complete the following steps:
+
+1. Connect the **Device 1** to the computer using a USB cable. The device is assigned a `COM` port (Windows) or `ttyACM` device (Linux), which is visible in the Device Manager.
```

Comment:
What about MAC os? 'tty.usbmodem' ... that is what I see nt sure if this is a given tough